source/gui/__init__.py

```diff
@@ -432,6 +445,17 @@ def __init__(self, frame):
 		# Translators: The label for the menu item to toggle Speech Viewer.
 		item=self.menu_tools_toggleSpeechViewer = menu_tools.AppendCheckItem(wx.ID_ANY, _("Speech viewer"))
 		self.Bind(wx.EVT_MENU, frame.onToggleSpeechViewerCommand, item)
+
+		# Translators: The label for the menu item to toggle Braille Viewer.

```

Comment:
I think this has to be moved two lines below. This is now causing checkpot to fail.